main.py

```diff
@@ -989,8 +989,6 @@ def get_redirect_route(
             constants.TASK_ENTITY_TYPE_EXPLORATION),
         improvements.ExplorationImprovementsConfigHandler),
 
-    get_redirect_route(
-        r'%s' % feconf.BLOG_ADMIN_PAGE_URL, blog_admin.BlogAdminPage),
```

Comment:
Can we remove `BLOG_ADMIN_PAGE_URL` from feconf? (Please check it's not used elsewhere before removal!)

Reply:
Done